### PR TITLE
Version of minecraft with persistent storage

### PR DESCRIPTION
**Pull request by deathlessdd**
Old minecraft sdl didnt have persistent storage so ive added it here.

### CODE REVIEW

**Comment by deathlessdd** [comment]
> If the other version didn't support, wouldn't make it more sense just make it support in the previous file ? I guess what I am saying is, shouldn't we have a single version with it? Maybe even make it clear that is optional using comments and pre-default it to one or another.

Well the issue that I had is the first sdl is for fun and testing to see it run on akash, It would confuse people vs having a dedicated one for actual server use. Which is why I didnt just add it to the original.

**Comment by joshlopes** [comment]
@deathlessdd I think it will confuse more people to have a "fun" and a "serious" one. If I'm playing around or testing for "fun" I want it to be close as a "serious" one, unless there is a reason for me not to.

I would prefer comments around the "fun" one, say to spec requirements to make it cheaper, in the SDL instead having to maintain two SDLs for the same thing. 

TL;DR: I think we should use only one, with comments if needed around stuff that we can disable/enable for fun/seriousness.
